http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala

```diff
@@ -110,6 +120,31 @@ class NonLeoMessageSubscriber[F[_]: Timer](gkeAlg: GKEAlgebra[F],
           }
         } yield ()
     } yield ()
+
+  private[monitor] def handleDeleteNodepoolMessage(
+    msg: DeleteNodepoolMessage
+  )(implicit ev: Ask[F, AppContext]): F[Unit] =
+    for {
+      ctx <- ev.ask
+      task = gkeAlg.deleteAndPollNodepool(DeleteNodepoolParams(msg.nodepoolId, msg.googleProject)).onError {
+        case _ =>
+          for {
+            _ <- logger.error(
+              s"An error occurred during deletion of nodepool ${msg.nodepoolId} in project ${msg.googleProject}. | trace id: ${ctx.traceId}"
```

Comment:
I don't think you need to log here. `AsyncTaskProcessor` will log error

Reply:
so should I remove the `.onError` entirely?

Reply:
yea, I think so

Reply:
sounds good will do

Reply:
just updated @Qi77Qi!